deployer/scripts/upgrade.sh

```diff
@@ -278,20 +286,103 @@ function getArrayIndex() {
   fi
 }
 
+# This function will wait until the latestVersion changes
+function waitForChange() {
+
+  local currentVersion=$1
+  local dc=$2
```

Comment:
you're right, sorry, was thinking of metadata latest version which would be
different

On Tue, May 24, 2016 at 10:44 AM, Eric Wolinetz <notifications@github.com>
wrote:

> In deployer/scripts/upgrade.sh
> <https://github.com/openshift/origin-aggregated-logging/pull/141#discussion_r64404496>
> :
>
> > @@ -278,20 +286,103 @@ function getArrayIndex() {
> >    fi
> >  }
> >
> > +# This function will wait until the latestVersion changes
> > +function waitForChange() {
> > +
> > +  local currentVersion=$1
> > +  local dc=$2
>
> From what I've seen only DC have a status.latestVersion
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/141/files/67600e358fe2e3fe7259c43a484b1b4378b3ac48#r64404496>
>